seacrowd/sea_datasets/idk_mrc_nli/idk_mrc_nli.py

```diff
@@ -0,0 +1,213 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+The IDKMRC-NLI dataset is derived from the IDK-MRC \
+    question answering dataset, utilizing named \
+    entity recognition (NER), chunking tags, \
+    Regex, and embedding similarity techniques \
+    to determine its contradiction sets. \
+    Collected through this process, \
+    the dataset comprises various columns beyond \
+    premise, hypothesis, and label, including \
+    properties aligned with NER and chunking tags. \
+    This dataset is designed to facilitate Natural\
+    Language Inference (NLI) tasks and contains \
+    information extracted from diverse sources \
+    to provide comprehensive coverage. Each data \
+    instance encapsulates premise, hypothesis, label, \
+    and additional properties pertinent to NLI evaluation.
+"""
+import csv
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Tasks, Licenses
+
+# The workshop submission at 18 April. I will change this _CITATION on that day.
+_CITATION = """\
```

Comment:
Hi @muhammadravi251001 , could you please add the` _CITATION`?

Reply:
For the `_CITATION`, because we are changing the publication's place, the acceptance notification is on July 5, 2024, Sir.

Reply:
After I checked each of the `.csv` files and tested it with `.load_dataset()` function; I didn't get any difference in a number of samples.

This is for `source` schema:
```
from datasets import load_dataset

data = load_dataset("seacrowd/sea_datasets/idk_mrc_nli/idk_mrc_nli.py", name="idk_mrc_nli_source")
data
```

```
DatasetDict({
    train: Dataset({
        features: ['premise', 'hypothesis', 'label'],
        num_rows: 18664
    })
    test: Dataset({
        features: ['premise', 'hypothesis', 'label'],
        num_rows: 1688
    })
    validation: Dataset({
        features: ['premise', 'hypothesis', 'label'],
        num_rows: 1528
    })
})
```

This is for `seacrowd` schema:
```
from datasets import load_dataset

data = load_dataset("seacrowd/sea_datasets/idk_mrc_nli/idk_mrc_nli.py", name="idk_mrc_nli_seacrowd_pairs")
data
```

```
DatasetDict({
    train: Dataset({
        features: ['id', 'text_1', 'text_2', 'label'],
        num_rows: 18664
    })
    test: Dataset({
        features: ['id', 'text_1', 'text_2', 'label'],
        num_rows: 1688
    })
    validation: Dataset({
        features: ['id', 'text_1', 'text_2', 'label'],
        num_rows: 1528
    })
})
```

Reply:
Ooh, I get the issue now, the `README.md` file on my [homepage](https://huggingface.co/datasets/muhammadravi251001/idkmrc-nli) it seems like I have one data surplus on there.

I forgot to decrease it by 1 to ignore the column row.

I've already edited my `README.md` there. Thanks for the info, Sir!